test/functional/functional_connection_test.rb

```diff
@@ -9,6 +9,19 @@ def test_opening_and_closing_connection
     assert_valid_closed_connection(connection)
   end
 
+  def test_connection_client_info
+    connection = Vertica::Connection.new(TEST_CONNECTION_HASH)
+    connection.query("SELECT client_type, client_os, client_version, client_label FROM SESSIONS WHERE client_pid = "+Process.pid.to_s) do |row|
```

Comment:
- Again, I'd use string interpolation here. 
- And to set the right example: `Vertica.quote(Process.pid)`